infrastructure/cdn-in-a-box/readiness/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM alpine:3.11
+
+RUN apk add --no-cache --update \
+  curl \
+  bind-tools \
+  net-tools \
+  jq \
+  bash
+
+# MANIFEST
+# to-access.sh (sourced, get to-get and env vars)
+# run.sh       (wait on TO, then to-get deliveryservices, then curl the exampleURLs)
+COPY ./infrastructure/cdn-in-a-box/traffic_ops/to-access.sh /opt/readiness/
+COPY ./infrastructure/cdn-in-a-box/readiness/run.sh /opt/readiness/
```

Comment:
You can do this in one line to save a cache layer: `COPY file [files...] destination`